modules/ROOT/pages/design-best-practices.adoc

```diff
@@ -0,0 +1,117 @@
+= Best Practices when using API Designer
+
+When you perform tasks on Design Center, you may encounter issues. The following topics help you resolve them:
+
+== General Best Practices
+
+* Divide your fragments into shorter reusable dependencies when possible
+
+* Have one version of the same dependency
+
+* Have only one browser tab open at a time for API Designer
+
+=== API Designer Resources
+
+When using API Designer, the resources to parse the specification depend on the user. Having a computer with enough available resources is crucial for API Designer to function properly.
+
+Suggested resources:
+
+* 8GB RAM
+* Ryzen series 5 / Intel i5 GPU or higher
+
+Due to parser limitations, your API Designer projects should have a total project size between 500 Kilobytes (KB) and 1.5 Megabytes (MB). 
```

Comment:
I would not say between 500KB and 1.5MB, I would just mention that project total size should be ideally below 500MB, but that the limit is on 1.5MB.
Because if not users may understand that their project size must be above 500KB when having projects below 500KB is ideal.

Reply:
Also, would clarify that project size is considered by the addition of all files in the project